mesheryctl/internal/cli/root/workspaces/workspace.go

```diff
@@ -0,0 +1,106 @@
+// Copyright 2024 Layer5, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package workspaces
+
+import (
+	"fmt"
+
+	"github.com/layer5io/meshery/mesheryctl/internal/cli/root/config"
+	"github.com/layer5io/meshery/mesheryctl/internal/cli/root/system"
+	"github.com/layer5io/meshery/mesheryctl/pkg/utils"
+
+	"github.com/fatih/color"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var (
+	name        string
+	description string
+	orgID       string
+	page        int
+
+	maxRowsPerPage       = 25
+	whiteBoardPrinter    = color.New(color.FgHiBlack, color.BgWhite, color.Bold)
+	availableSubcommands = []*cobra.Command{listWorkspaceCmd, CreateWorkspaceCmd}
+)
+
+var WorkSpaceCmd = &cobra.Command{
+	Use:   "workspace",
+	Short: "View list of workspace and detailed information of a specific relationship",
+	Long:  "View list of workspace and detailed information of a specific relationship",
```

Comment:
What is difference between long and short again? you can copy text from docs.layer5.io for long text